src/components/DpTooltip/DpTooltip.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpTooltip from './DpTooltip.vue'
+
+const meta: Meta<typeof DpTooltip> = {
+    component: DpTooltip,
+    title: 'Components/Tooltip',
+    render: (args) => ({
+        components: {
+            DpTooltip,
+        },
+        setup() {
+            return { args }
+        },
+        template: `<dp-tooltip v-bind='args'>
+          my text
+        </dp-tooltip>`,
+    })
+}
+
+interface IDpAccordion {
+    text: string
+}
+
+type Story = StoryObj<IDpAccordion>
+
+export default meta
+
+export const Default: Story = {
+    args: {
+        text: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
```

Comment:
I'd suggest to keep both `text` values in sync (here, and .stories.tsx).

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/505/commits/c16ba2ff8249ebc93299ef697e668d5bc5331534